src/main/java/dinkplugin/notifiers/data/CollectionNotificationData.java

```diff
@@ -18,4 +26,23 @@ public class CollectionNotificationData extends NotificationData {
     @Nullable
     Long price;
 
+    @Nullable
+    Integer completedEntries;
+
+    @Nullable
+    Integer totalEntries;
+
+    @Override
+    public List<Field> getFields() {
+        if (completedEntries != null && totalEntries != null) {
+            String percent = PERCENT_FORMAT.get().format(1.0 * completedEntries / totalEntries);
+            return Collections.singletonList(
+                new Field(
+                    "Completed Entries",
+                    Field.formatBlock("", String.format("%d/%d (%s)", completedEntries, totalEntries, percent)))
```

Comment:
could we maybe just use String.format when formatting the % instead?
`%2.f` or something

Reply:
Sure. String format forces half-up rounding (was using DecimalFormat to do half-even), but this isn't a big deal & simplifies the code